### PR TITLE
Update bug templates

### PR DESCRIPTION
**Pull request by raphj**
This MR updates the bug templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -5,71 +5,54 @@ title: "Bug:"
...
 ## Screenshots
 
-*Optional - add screenshots to help explain your problem.*
+Help us understand the problem with screenshots. You can use [peek](https://github.com/phw/peek) to generate animated GIFs.
 
-*Please consider sharing an [animated GIF screencast generated with peek](https://github.com/phw/peek)*
 
-## Impacted components
+*The following sections are for the maintainers*
+________________________________________________
```
Comment (gbonaspetti) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: Maybe the line should be before the phrase, what do you think?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -5,71 +5,54 @@ title: "Bug:"
...
 
-- *backend*
-- *frontend*
-- *htmldoc / file / thread app*
-- *webdav*
-- *...*
+## Prerequisites
 
-*If unsure, write "unknown"*
+Optional - list the issues that must be solved before handling this issue.
```
Comment (gbonaspetti) [inline .github/ISSUE_TEMPLATE/bug_report.md:49]:
```suggestion
Optional - list the issues that must be solved or what needs to be done before handling this issue.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -6,35 +6,18 @@ labels: to sort
...
-*For example:*
-
-- *backend*
-- *frontend*
-- *htmldoc / file / thread app*
-- *webdav*
-- *...*
-
-*If unsure, write "unknown"*
+Optional - list the issues that must be solved or what needs to be done before handling this ticket.
```
Comment (gbonaspetti) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]:
```suggestion
Optional - list the issues that must be solved or what needs to be done before handling this issue.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (raphj) [reply .github/ISSUE_TEMPLATE/bug_report.md:41]: ok for me :-)

**Pull request merged**